src/composables/sdk.ts

```diff
@@ -0,0 +1,39 @@
+import type {
+  IDefaultComposableOptions,
+} from '../types';
+import { useAccounts } from './accounts';
+import { IProtocol } from '../lib/protocols/IProtocol';
+import { AeternityProtocol } from '../lib/protocols/AeternityProtocol';
+import { BitcoinProtocol } from '../lib/protocols/BitcoinProtocol';
+import { DogecoinProtocol } from '../lib/protocols/DogecoinProtocol';
+
+let sdkRegistry: Record<string, IProtocol>;
+
+export function useSdk({ store }: IDefaultComposableOptions) {
+  const { activeAccount } = useAccounts({ store });
+
+  sdkRegistry = {
+    aeternity: new AeternityProtocol(store),
+    bitcoin: new BitcoinProtocol(),
+    dogecoin: new DogecoinProtocol(),
```

Comment:
The `keys` could be derived from constants. hardcoded here just to get the approach reviewed.